CONTRIBUTING.md

```diff
@@ -45,8 +45,19 @@ The usual contributing steps are:
 
 ### Contributing to the documentation.
 
-Gatsby, unsurprisingly uses Gatsby for it's documentation website. Here are the
-steps to add/modify documentation and preview it.
+Gatsby, unsurprisingly, uses Gatsby for it's documentation website.
+
+If you want to add/modify some documentation markdown to make a change in the
+website, go to the
+[docs folder on Github](https://github.com/gatsbyjs/gatsby/tree/master/docs) and
+use the inline editor to edit and then preview your changes. Github then allows
```

Comment:
I think it's called the "file editor" in [Github](https://help.github.com/articles/editing-files-in-your-repository/) rather than "inline editor," unless I am mistaken!

Reply:
I dont know too! But I'll trust you on this.